fastapi_soap/wsdl.py

```diff
@@ -69,18 +65,18 @@ def generate_xsd_element(
         )
         element.set('type', f'xs:{element_type}')
 
-    if model_field and model_field.is_complex():
-        if model_field.shape == 2:
-            # TODO: get gt and lt
-            element.set(
-                'minOccurs', str(model_field.field_info.min_items or 0)
-            )
-            element.set(
-                'maxOccurs',
-                str(model_field.field_info.max_items or 'unbounded'),
-            )
-        else:
-            ...
+    # if model_field and model_field.is_complex():
+    #     if model_field.shape == 2:
+    #         # TODO: get gt and lt
+    #         element.set(
+    #             'minOccurs', str(model_field.field_info.min_items or 0)
+    #         )
+    #         element.set(
+    #             'maxOccurs',
+    #             str(model_field.field_info.max_items or 'unbounded'),
+    #         )
+    #     else:
```

Comment:
i did it [this way](https://github.com/mittmannv8/fastapi-soap/pull/9/files#diff-4add8ea3d5633ce7ecbdceaa3468350a36505fccc636f08b5e6fb9cae527cb99R68-R73)